misk/src/main/kotlin/misk/web/jetty/JettyService.kt

```diff
@@ -243,6 +248,25 @@ class JettyService @Inject internal constructor(
 
     server.handler = statisticsHandler
 
+    webConfig.cors.forEach { (path, corsConfig) ->
+      val holder = FilterHolder(CrossOriginFilter::class.java)
+      holder.setInitParameter(CrossOriginFilter.ALLOWED_ORIGINS_PARAM,
+          corsConfig.allowedOrigins.joinToString(","))
+      holder.setInitParameter(CrossOriginFilter.ALLOWED_HEADERS_PARAM,
+          corsConfig.allowedHeaders.joinToString(","))
+      holder.setInitParameter(CrossOriginFilter.ALLOWED_METHODS_PARAM,
```

Comment:
mind ordering these statements the same as they’re declared in the config object? This one has [origins, headers, methods...] and the config has [origins, methods, headers...] 

It’s easier to review & maintain if the orders are consistent! (Though I have no preference of order)